hmda/src/main/scala/hmda/validation/rules/lar/validity/V619_2.scala

```diff
@@ -0,0 +1,18 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.filing.lar.LoanApplicationRegister
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateSyntax._
+import hmda.validation.dsl.ValidationResult
+import hmda.validation.rules.EditCheck
+
+object V619_2 extends EditCheck[LoanApplicationRegister] {
+  override def name: String = "V619-2"
+
+  override def parent: String = "V619"
+
+  override def apply(lar: LoanApplicationRegister): ValidationResult = {
+    lar.action.actionTakenDate.toString.slice(0, 4) is equalTo("2018")
```

Comment:
Should we put this in configuration file, able to override with environment variable? What happens in 2019?